### PR TITLE
feat(analytics): date range in analytics filters

### PR DESCRIPTION
**Pull request by jyash97**
### What does this PR do:
https://www.notion.so/appbase/Arc-Restrict-analytics-access-for-more-than-30-days-2a3b4dddea8041cb87ec84aa855a22cc

### Testing: 
https://www.loom.com/share/4298b96f27b94c44a8af223825bee391

### Pages Affected
Analytics pages with Filters

### CODE REVIEW

File: components/analytics/components/Filter/index.js [hunk 1]
```diff
@@ -11,30 +11,64 @@ import { isValidPlan } from '../../../../utils';
...
-	weekly: {
-		from: moment()
-			.subtract(7, 'days')
-			.format('YYYY/MM/DD'),
+	'Last Week': {
+		from: moment().subtract(14, 'days').format('YYYY/MM/DD'),
+		to: moment().subtract(7, 'days').format('YYYY/MM/DD'),
+	},
+	'This Month': {
+		from: moment().subtract(30, 'days').format('YYYY/MM/DD'),
```
[the remark this answers is not in the document]
Comment (bietkul) [reply components/analytics/components/Filter/index.js:24]: It should be {start of the current month} to {current date}

File: components/analytics/components/Filter/index.js [hunk 2]
```diff
@@ -11,30 +11,64 @@ import { isValidPlan } from '../../../../utils';
...
+		to: moment().format('YYYY/MM/DD'),
+	},
+	'Last Month': {
+		from: moment().subtract(60, 'days').format('YYYY/MM/DD'),
```
Comment (bietkul) [inline components/analytics/components/Filter/index.js:28]: @jyash97 That is wrong too.

File: components/analytics/components/Filter/index.js [hunk 3]
```diff
@@ -11,30 +11,64 @@ import { isValidPlan } from '../../../../utils';
...
 const { Option } = Select;
 
-const dataRangeFilters = {
-	monthly: {
-		from: moment()
-			.subtract(30, 'days')
-			.format('YYYY/MM/DD'),
+const dateRangeFilters = {
+	'This week': {
+		from: moment().subtract(7, 'days').format('YYYY/MM/DD'),
```
Comment (bietkul) [inline components/analytics/components/Filter/index.js:16]: @jyash97 That is wrong too.
It should be {start of the current week} to {current date}

File: components/analytics/components/Filter/index.js (hunk 2 above)
Comment (bietkul) [reply components/analytics/components/Filter/index.js:28]: It should be {start of the last month} to {end of last month}

**Comment by jyash97** [comment]
Changes merged in  #214

**Pull request closed without merging**